user-interface/README.md

```diff
@@ -2,6 +2,61 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Not necessary for you to fix here, but since we got rid of create-react-app, we should probably remove this.

Reply:
That was going to be my question to the team.